azure-simple/packer/ansible-ei/site.yml

```diff
@@ -0,0 +1,29 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
```

Comment:
@rasanjaya85 please update the license header year.

Reply:
Done